R/utilities.R

```diff
@@ -3571,3 +3570,37 @@ cleanup_maf = function(maf_df){
 
   return(maf_df)
 }
+
+#' Complement maf with missing samples.
+#'
+#' @param incoming_maf The initial MAF data frame to be supplemented with missing samples.
+#' @param these_samples_metadata The metadata data frame that contains Tumor_Sample_Barcode column with ids to be present in the complemented MAF.
+#'
+#' @return maf_df with complemented Tumor_Sample_Barcode and other columns ready to be used downstream
+#' @export
```

Comment:
Not necessarily, because the dplyr is already in the `NAMESPACE` and will be installed with GAMBLR. Referring to the functions here with `dplyr::` let's avoid the import. But I would definitely add if this was a new package, to get it added as dependency